docs/source/starter/introduction.rst

```diff
@@ -12,103 +12,58 @@
 
 .. _new_project:
 
-############
-Introduction
-############
 
-**************************
-What is PyTorch Lightning?
-**************************
+####################
+Lightning in 2 Steps
+####################
 
-PyTorch Lightning provides you with the APIs required to build models, datasets, and so on. PyTorch has all you need to train your models; however, there’s much more to deep learning than attaching layers. When it comes to the actual training, there’s a lot of boilerplate code that you need to write, and if you need to scale your training/inferencing on multiple devices/machines, there’s another set of integrations you might need to do.
+**In this guide we'll show you how to organize your PyTorch code into Lightning in 2 steps.**
 
-PyTorch Lightning solves these for you. All you need is to restructure some of your existing code, set certain flags, and then you are done.
-Now you can train your models on different accelerators like GPU/TPU/IPU, to do distributed training across multiple machines/nodes without code changes using state-of-the-art distributed training mechanisms.
+Organizing your code with PyTorch Lightning makes your code:
 
-Code organization is the core of Lightning. It leaves the research logic to you and automates the rest.
-
-----------
-
-********************
-Lightning Philosophy
-********************
-
-Organizing your code with Lightning makes your code:
-
-* Flexible (this is all pure PyTorch), but removes a ton of boilerplate
+* Keep all the flexibility (this is all pure PyTorch), but removes a ton of boilerplate
 * More readable by decoupling the research code from the engineering
 * Easier to reproduce
 * Less error-prone by automating most of the training loop and tricky engineering
 * Scalable to any hardware without changing your model
 
-Lightning is built for:
-
-* Researchers who want to focus on research without worrying about the engineering aspects of it
-* ML Engineers who want to build reproducible pipelines
-* Data Scientists who want to try out different models for their tasks and build-in ML techniques
-* Educators who seek to study and teach Deep Learning with PyTorch
```

Comment:
`Focus on research and leave the engineering to us` is the highlight, but does that mean that lightning is only for researchers? aren't we missing potential users here?

Reply:
> Focus on research and leave the engineering to us

@rohitgr7 Where is this highlight?